### PR TITLE
Up and down vote

### PR DESCRIPTION
**Pull request by samcole1992**
Completed work for upvote/downvotes in React for Reviews.

### CODE REVIEW

File: app/controllers/api/v1/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  #     review.save
+  #     @reviews = Review.order(:id)
+  #     render json: @reviews
+  #   elsif data["type"] == "down_votes"
+  #     review.down_votes += 1
+  #     review.save
+  #     @reviews = Review.order(:id)
+  #     render json: @reviews
+  #   end
+  # end
```
Comment (jennceng) [inline app/controllers/api/v1/reviews_controller.rb]: what’s all this commented out code in reviews_controller?

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -2,8 +2,13 @@ class ReviewsController < ApplicationController
 
   def index
     @reviews = Review.all
+    respond_to do |format|
+      format.html
+      format.json { render json: @reviews}
+    end
```
Comment (jennceng) [inline app/controllers/reviews_controller.rb]: seems a bit redundant to have both this and an api for reviews

File: app/controllers/api/v1/reviews_controller.rb (hunk 1 above)
Comment (samcole1992) [inline app/controllers/api/v1/reviews_controller.rb]: Deleted the commented code

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (samcole1992) [inline app/controllers/reviews_controller.rb]: got rid of this

**Pull request merged**